modules/admin-ui/src/main/webapp/scripts/modules/events/controllers/serieController.js

```diff
@@ -207,24 +207,36 @@ angular.module('adminNg.controllers')
             });
         });
 
-        $scope.theme = {};
+        $scope.selectedTheme = {};
+
+        $scope.updateSelectedThemeDescripton = function () {
+            if(angular.isDefined($scope.themeDescriptions)) {
```

Comment:
Not sure about the spacing here. I definitely have seen both, `if(` and `if (`, but the latter seems prevalent.